PyInstaller/building/datastruct.py

```diff
@@ -14,8 +14,8 @@
 
 from PyInstaller.utils import misc
 from PyInstaller.utils.misc import load_py_data_struct, save_py_data_struct
```

Comment:
Looks like I missed another `misc` & `load_py_data_struct`/`save_py_data_struct` combo in the previous review - sorry!